lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -4084,8 +4087,8 @@ data ApiMintData (n :: NetworkDiscriminant) = ApiMintData
         -- If no address is specified, then minted assets will be returned to
         -- the wallet as change, and change output addresses will be assigned
         -- automatically.
-    , amount
-        :: Quantity "assets" Natural
+    , quantity
+        :: Natural
```

Comment:
Hi @paweljakubas. Is this a breaking change to the API?

Reply:
well, yes. but new workflow is still experimental API ;-)